lib/engine/game/g_18_eus/game.rb

```diff
@@ -99,8 +110,8 @@ class Game < Game::Base
         ].freeze
 
         TRAINS = [
-          { name: '2', distance: 2, price: 100, rusts_on: '4', num: 20 },
-          { name: '2+', distance: 2, price: 100, obsolete_on: '4', num: 10 },
+          { name: '2', distance: 2, price: 100, rusts_on: '4', num: 2 },
```

Comment:
Typo?  We need more then 2 2-trains...

Reply:
Testing code that I fogot to revert. Good catch.